src/panels/config/ha-panel-config.ts

```diff
@@ -393,6 +403,9 @@ class HaPanelConfig extends SubscribeMixin(HassRouterPage) {
       subscribeLabelRegistry(this.hass.connection!, (labels) => {
         this._labelsContext.setValue(labels);
       }),
+      subscribeFloorRegistry(this.hass.connection!, (floors) => {
+        this._floorsContext.setValue(floors);
```

Comment:
Since floors is now in `hass`, we can do this on a higher level in the `context-mixin`, allowing elements to use floors without needing access to `hass`.